tabbycat/privateurls/views.py

```diff
@@ -149,12 +156,12 @@ class EmailRandomisedUrlsView(RoleColumnMixin, TournamentTemplateEmailCreateView
 
     tournament_redirect_pattern_name = 'privateurls-list'
 
-    def get_extra(self):
+    def get_extra(self) -> Dict[str, Union[int, str]]:
```

Comment:
I think I'd use `Dict[str, Any]` here as we don't care that it's only `int` or `str`.